Main.java

```diff
@@ -9,7 +11,24 @@
    * @param args the command line arguments
    */
   public static void main(String[] args) {
-    // TODO code application logic here
-    
+    // Creat a Scanner for user input
+    Scanner input = new Scanner(System.in);
+    
+    // Asks the users how and what they want to divide
+    System.out.println("Please enter in two integers, on seperate lines, to divide.");
+
+    // lets the users input numbers for the answers
+    int x1 = input.nextInt();
+    int x2 = input.nextInt();
+
+    // get the users answers
+    int answer = x1 % x2;
+    int answer2 = x1 / x2;
```

Comment:
While the program works, these variable names are not really descriptive. x1 and x2 are probably ok because they are just some numbers but answer and answer2 would be better named remainder and quotient, as that is what they really are. Just remember, a variable's name should describe what it is storing or being used for. Otherwise, this is good looking code.